### ISSUE TITLE
Keyboard Interaction for a stack view component

### ISSUE THREAD

**Issue by dkanev** [vmware/clarity#2357]
[X ] feature request
[X ] enhancement

I would like to request a Keyboard Interaction for the stack view component
Probably they should be something like the on specified by the “WAI-ARIA Authoring Practices 1.1” for Accordion (https://www.w3.org/TR/wai-aria-practices/#accordion)
For example:
The stack view is good to support collapsed and expanded with a keyboard. 

We have a lot of Angular JS components which use the downgrade stack block. We would like if possible the above requested improvements to be made in such a way that to be available in the downgraded stack blocks.

Environment details
Angular version: 4.x.x

Clarity version: 11.0.1

OS and version:**

Browser:all

**Comment by dkanev** [comment]
We have a lot of Angular JS components which uses the downgrade stack block. We would like if possible the above requested improvements to be made in such a way that to be available in downgraded stack blocks.

**Comment by mathisscott** [comment]
Our plan is to enable this via tabindex. This should allow users to interact with stack views via the keyboard.
